draft-ietf-ppm-dap.md

```diff
@@ -2341,6 +2371,10 @@ Upon receipt of a `CollectionJobReq`, the Leader begins by checking that it
 recognizes the task ID in the request path. If not, it MUST abort with error
 `unrecognizedTask`.
 
+Next, the Leader checks that the aggregation parameter is valid as described in
+{{agg-param-validation}}. If the aggregation parameter is invalid, then the
+Leader MUST abort with error `invalidAggregationParameter`.
```

Comment:
I think we should discuss this either above in L2350-2352 or here, but not in both places. It's confusing for implementers to have the same MUST laid out twice.

Reply:
Just above, the first MUST is to remind the Collector that the agg param needs to be valid. Here we're telling the Leader what to do if the agg param is invalid.

Reply:
I wasn't sure if a MUST on both the sender (Collector) and receiver (Leader) side is necessary, but re-reading RFC 2119, I think it's appropriate. Let's keep what you have here.